docs/Installation-Windows.md

```diff
@@ -7,7 +7,7 @@ Windows VM such as Bootcamp or Parallels.
 
 To use the ML-Agents toolkit, you install Python and the required Python
 packages as outlined below. This guide also covers how set up GPU-based training
-(for advanced users). GPU-based training is not required for the v0.4 release of
+(for advanced users). GPU-based training is not required for the v0.6 release of
```

Comment:
On second thought, can we just remove v0.6 and say "for the current release"

Reply:
More appropriately .... "GPU-based training is not currently required for the ML-Agents toolkit."

Reply:
Fixed.